code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -383,6 +384,47 @@
 		M.AdjustCultSlur(20 SECONDS) //CUASE WHY THE HELL NOT
 	return ..() | update_flags
 
+/datum/reagent/fuel/unholywater/overdose_process(mob/living/M, severity)
```

Comment:
my two cents is that an overdose process is unnecessary---stick with lighter  changes, first.